server/keycloak.js

```diff
@@ -260,6 +260,11 @@ class Keycloak {
     }
   }
 
+  getUserUrl(userId = '') {
+    if (!userId) throw new Error('keycloak: User ID is required');
+    return `${this.authUrl}/admin/realms/${this.realm}/users/${userId}`;
```

Comment:
I think we're still performing requests from user supplied data here, but putting it in a function is good to manage this scenario consistently. Is there additional validation that we can do on the userId? Or would that just open up opportunities for SQL injection?

Reply:
actual data is not user-supplied. It is from DB. How do I explain this to the scanner?

Reply:
And this is not an SQL query .... this is KeyCloak URL which has sufficient protection